### PR TITLE
Policheck

### PR DESCRIPTION
**Pull request by reyang**

### CODE REVIEW

**Review by Kielek — APPROVED** [review]

File: docs/diagnostics/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ to specific code patterns identified through static analysis.
 
 ## Experimental APIs
 
-Range: OTEL1XXX
+Range: OTEL1000 - OTEL1999
```
Comment (alanwest) [inline docs/diagnostics/README.md:10]: This change seems to make things more clear, but What's Policheck? Was OTEL1XXX caught by some lint check or something?

**Review by alanwest — APPROVED** [review]
LGTM, one question though...

File: docs/diagnostics/README.md (hunk 1 above)
Comment (reyang) [reply docs/diagnostics/README.md:10]: > What's Policheck?

It is a tool used inside Microsoft to check potential offensive/sensitive words in codebase.

> Was OTEL1XXX caught by some lint check or something?

It was discovered by a Microsoft product which takes OpenTelemetry .NET source code. The story is similar to https://bugs.chromium.org/p/chromium/issues/detail?id=981129.

Comment (alanwest) [reply docs/diagnostics/README.md:10]: So, you're saying OpenTelemetry .NET is now rated PG rather than XXX? 😆

**Pull request merged**